code/api/src/modules/product/types.js

```diff
@@ -1,5 +1,5 @@
 // Imports
-import { GraphQLObjectType, GraphQLString, GraphQLInt } from 'graphql'
+import { GraphQLObjectType, GraphQLString, GraphQLInt, GraphQLBoolean } from 'graphql'
```

Comment:
GraphQLBoolean is required since it's a new type that hasn't been used in the other properties